modules/spring-config/src/test/java/org/mule/runtime/config/spring/extension/schema/ModuleSchemaGeneratorTestCase.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.spring.extension.schema;
+
+import static java.util.stream.Collectors.toList;
+import org.mule.runtime.config.spring.dsl.model.extension.ModuleExtension;
+import org.mule.runtime.config.spring.dsl.model.extension.loader.ModuleExtensionStore;
+import org.mule.runtime.config.spring.dsl.model.extension.schema.ModuleSchemaGenerator;
+import org.mule.runtime.core.util.IOUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+
+import org.custommonkey.xmlunit.DetailedDiff;
+import org.custommonkey.xmlunit.Diff;
+import org.custommonkey.xmlunit.Difference;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class ModuleSchemaGeneratorTestCase {
```

Comment:
It's not a mule type of test, just generates schemas (it does not lunch any mule instance). Does it make sense to extend from that class?